examples/phase/impingement/mesh40x1.py

```diff
@@ -256,3 +256,8 @@
     exec(fipy.tests.doctestPlus._getScript())
 
     raw_input('finished')
+
+#  LocalWords:  WarrenPolycrystal ModularVariable et al Acta Materialia mPhiVar
```

Comment:
Hrmmmm...

Not sure I approve of cluttering up our code with this sort of thing. There's no way to add a dictionary to the repo?

Reply:
Oh, that's an erroneous include, sorry. `emacs` must have barfed on exit.